spaceflights/{{ cookiecutter.repo_name }}/src/requirements.txt

```diff
@@ -1,14 +1,16 @@
 black~=22.0
-flake8>=3.7.9, <4.0
+flake8>=3.7.9, <5.0
 ipython>=7.31.1, <8.0
 isort~=5.0
 jupyter~=1.0
+jupyterlab_server>=2.11.1, <2.16.0
 jupyterlab~=3.0
-kedro[pandas.CSVDataSet, pandas.ExcelDataSet, pandas.ParquetDataSet]~={{ cookiecutter.kedro_version }}
+kedro~=0.18.3
```

Comment:
Why is it now pinned to a specific version instead of following the kedro version?

Reply:
I've been agonising over this! I should have flagged it. @noklam 

I think we agreed we'd pin the tutorial to a specific version of Kedro @yetudada ? And because the tutorial is tightly bound to the starter (or the starter bound to the tutorial) then the starter should also be pinned to the Kedro version.

(which should be 18.4 by the way, but I only realised that after standup yesterday)

I am of the view that we should pin to ensure that tutorial/starter are completely aligned and that for _every release_ of Kedro, we test the starters and update them and re-pin to the new version (with changes to the documentation text too).  It's the only way to be sure we keep alignment and make the user experience nice. And if we don't have time to do it, at least by pinning to an earlier version of Kedro, we can be sure it'll still work for the reader.

Thoughts?

Reply:
We should not pin the starters. This is because we want the starters to work with **every** version of Kedro. If a user has Kedro `0.17.3` installed and then uses the starter it will install that exact same version of the starter. If the user has `0.18.4` it will use that version. By pinning Kedro in the starters this mechanism won't work anymore and users would be forced to update their Kedro version.